coral/README.md

```diff
@@ -30,6 +36,12 @@ You can also run `pnpm` in your console to get a list of all available scripts.
 - `test`: runs all tests one time
 - `tsc`: runs the TypeScript compiler
 
+### Usage: How to run the Coral in build
+
+1. in `/coral`, run `make enable-coral-in-springboot` (see our [Makefile](../Makefile))
+2. go to root and follow the [instructions how to run Klaw](../README.md#Install)
+3. Klaw will run in `http://localhost:9097` (note: it's `http` instead of `https`!) 
```

Comment:
No, I also got that. But I did not try that out yet and wanted to add the other infos to the docs before I forget it again :D But if I'm updating more infos to the mock apis, I can also take the time and add the cert part, that would make the PR complete.

Reply:
I would suggest we add this in a later PR, because there are changes needed in `core/src/main/resources/application.properties` for that (as far as I understood). I think these changes together with an update of the readme make a good PR in itself!